tests/test_calendar_grid.py

```diff
@@ -1,19 +1,20 @@
 import datetime
 
-import app.routers.calendar_grid as cg
+from app.routers import calendar_grid
```

Comment:
Why?

Reply:
If I'm not mistaken, Yam asked somewhere in previous comments.

Reply:
yaa gonzom correct